utils/emacs/README.md

```diff
@@ -0,0 +1,32 @@
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# Emacs plugin for Carbon
+
+For now, simply derives from cc-mode.
+
+Install:
+
+1. copy carbon-mode.el to lisp package path
+2. add to config
+
+```emacs-lisp
+(require 'carbon-mode)
```

Comment:
These instructions assume that `(push "~/.emacs.d/lisp" load-path)` is in your `.emacs` file, and so should instruct people to add it if it is not already there.